external/upstream/optking/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ else()
 
     ExternalProject_Add(optking_external
         BUILD_ALWAYS 1
-        URL https://github.com/AlexHeide/optking/tarball/hessian
+        URL https://github.com/psi-rking/optking/tarball/master
```

Comment:
you can replace `master` with a tag name or the short hash for the known stable commit. Otherwise there's a risk of users building a cutting edge version.